core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -156,6 +157,7 @@ public KafkaSchemaRegistry(SchemaRegistryConfig config,
                            + " node where all register schema and config update requests are "
                            + "served.");
     this.masterNodeSensor.add(m, new Gauge());
+    this.zKAclsEnabled = config.getBoolean(SchemaRegistryConfig.ZOOKEEPER_SET_ACL_CONFIG);
```

Comment:
It looks like everywhere this is used, you do `JaasUtils.isZkSecurityEnabled() && zKAclsEnabled`

Would it make sense to just define
```
this.zKAclsEnabled = JaasUtils.isZkSecurityEnabled()  && config.getBoolean(SchemaRegistryConfig.ZOOKEEPER_SET_ACL_CONFIG);
```